docs/userdocs/k8s/deploy-submarine.md

```diff
@@ -0,0 +1,70 @@
+
+
+# Deploy Submarine On K8s
+
+## Deploy Submarine using Helm Chart (Recommended)
+
+### Create images
+submarine server
+```bash
+./dev-support/docker-images/submarine/build.sh
+```
+
+submarine database
+```bash
+./dev-support/docker-images/database/build.sh
+```
+
+### install helm
+For more info see https://helm.sh/docs/intro/install/
+
+### Deploy Submarine Server, mysql
+You can modify some settings in ./helm-charts/submarine/values.yaml
+```bash
+helm install submarine ./helm-charts/submarine
+```
+
+### Delete deployment
+```bash
+helm delete submarine
+```
+
+FIXME: TF Operator / PyTorch Operator can be deployed by Helm Chart, correct?
+
+## Deploy Submarine Manually
```

Comment:
In the pre-version deploy by manual it aims to standalone and supports more K8s cluster in one submarine server. But in here, my understanding is manually deploy submarine in K8s with in-cluster mode.